packages/blade/src/components/NativeAria/NativeAria.native.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Pressable, Text, View } from 'react-native';
+import mapA11yProps from '../../utils/mapProps';
+import { CloseIcon, EyeIcon } from '../Icons';
+
+const CheckboxItem: React.FC<{ label: string; checked?: boolean; disabled?: boolean }> = ({
+  label,
+  checked,
+  disabled,
+}) => {
+  const [isChecked, setChecked] = React.useState(checked);
+
+  const toggleChecked = (): void => {
+    if (disabled) return;
+    setChecked((prev) => !prev);
+  };
+
+  return (
+    <>
+      {isChecked ? (
+        <EyeIcon color="action.icon.link.active" size="small" />
+      ) : (
+        <CloseIcon color="action.icon.link.active" size="small" />
+      )}
+      <Pressable
+        onPress={toggleChecked}
+        {...mapA11yProps({
+          accessibilityDisabled: disabled,
+          accessibilityRole: 'checkbox',
+          accessibilityChecked: isChecked ? 'true' : 'false',
+        })}
+        focusable
+      >
+        <Text>{label}</Text>
+      </Pressable>
+    </>
+  );
+};
+
+const NativeAria = (): React.ReactElement => {
+  return (
+    <>
+      <Text nativeID="id-group-label">Fruites</Text>
+      <View
+        {...mapA11yProps({
```

Comment:
I think we shall have a better demo which shows our workflow. 
* Code sharing
* One component that works on both web and native
* a11y props on the component that resolves based on the platform.

Basically we can have the common checkbox and show the `mapA11y` function with that. It'll give better understanding

Reply:
This function will be internal right? Why do we need to have a full showcase? Or do we make an internal storybook?
And this NativeAria.tsx example is just for development purposes.